CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Open Trip Planner - React Redux Changelog
+
+## [3.0.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.0.0) (2021-04-13)
+
+- Add mobile batch results screen
+- User Account Settings
+- Add more buttons to favorites management
+- Re-organize user settings
+- Add field trip module
+
+## [3.1.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.1.0) (2021-07-07)
+
+- Add reverse directions button
+- Add mailables submodule to calltaker module
+- Improve map performance
+
+## [3.2.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.2.0) (2021-07-15)
+
+- Add GTFS-RT vehicle overlay
+- Add user language settings
+- Support overriding strings
+
+## [3.3.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.3.0) (2021-07-22)
+
+- Field trip module
+- Add save button
+- Add delete button
```

Comment:
Are these specific to the field trip module or something else?

Reply:
These are specific to the field trip module, these were meant to be indented, which I fixed in https://github.com/opentripplanner/otp-react-redux/pull/537/commits/5e363e393de0502c385885d4aefd95d2d61cc484